migrations/Version20230823111023.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230823111023 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf('mysql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE kiwi_location ADD name VARCHAR(255) NOT NULL');
```

Comment:
Am I correct to infer that this migration will leave all values in this new column empty? If so, that seems like a potential UX issue to me. I'd suggest copying the existing addresses as names as well during migration.

Reply:
Valid point, I've just copied the current location text into the new name col